docs/rfc/rfc-1.adoc

```diff
@@ -0,0 +1,102 @@
+:toc: macro
+
+= RFC 1: tBTCv2 Design
+
+:icons: font
+:numbered:
+toc::[]
+
+== Overview
+
+=== Minting
+
+Stakers periodically create https://en.bitcoin.it/wiki/BIP_0032[Hierarchical
+Deterministic Wallets] to hold frozen BTC assets to maintain the peg.
+Depositors send BTC funds to the most-recently-created-wallet at a secret
+derivation path. Then, the depositor hashes the combination of their secret
+derivation path and ethereum address and publishes that to the ethereum
+blockchain (a commitment). Once the commitment is published, the depositor
+reveals their secret derivation path and ethereum address. The tbtc system
+verifies that the reveal matches the commitment and that there UTXOs at the
+specified secret derivation path. If everything checks out, the bitcoin wallet
+moves the funds from the secret derivation path to the public derivation path
+and mints the equivalent amount of tBTCv2 at the revealed ethereum address.
```

Comment:
To move funds from the address created from the derivation path, do I need to know the private key for the account under that path? If so, I'd assume I also need to know the root private key and we do not know it - it's owned by a threshold singing group. 🤔 Can I do signing without constructing the account's private key so that I can sweep/redeem from the given path?